worlds/hexcellsinfinite/options.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any
+from dataclasses import dataclass
+from worlds.AutoWorld import PerGameCommonOptions
+from options import Choice, OptionGroup, Toggle
```

Comment:
```suggestion
from Options import Choice, OptionGroup, Toggle
```
Here your issue is that this import from the base `Options.py` shouldn't have been changed. But for a failed world load you should just try to generate locally and see what the error is.